### ISSUE TITLE
What's wa and wb？

### ISSUE THREAD

**Issue by Sarah20187** [3DCVdeveloper/BGNet#4]
Thank you for your wonderful work!
Could you explain that what are the learnable parameters wa and wb? It seems like they are not mentioned in the original paper?
https://github.com/3DCVdeveloper/BGNet/blob/19b9c1bc17ebf653b344e52470ae6f85953c17a9/models/bgnet_plus.py#L109

**Comment by 18651626101** [comment]
> Thank you for your wonderful work!
> Could you explain that what are the learnable parameters wa and wb? It seems like they are not mentioned in the original paper?
> https://github.com/3DCVdeveloper/BGNet/blob/19b9c1bc17ebf653b344e52470ae6f85953c17a9/models/bgnet_plus.py#L109

I also have the same question before. However, when I have studied deeply, I find that this is to generate the horizontal and vertical coordinate of the interpolation of bilateral grid. This is my personal opinion, welcome to discuss with you.

**Comment by Xinzhe99** [comment]
> 

i don't think so ,what you say may be hg and wg ? i also don't know what's the meaning of wa and wb
